aiohttp_json_rpc/client.py

```diff
@@ -156,3 +156,57 @@ def add_methods(self, *methods):
             del self._handler[topic]
 
         return await self.call('unsubscribe', params=topic, timeout=timeout)
+
+
+class JsonRpcMagicClientMethod:
+    """JSON-RPC callable jwaitable method representation.
+
+    Intended usage:
+
+    >>> rpc = JsonRpcClient()
```

Comment:
I think the variable name "rpc" is confusing because we use "rpc" for ``aiohttp_json_rpc.JsonRpc`` objects in the README

Reply:
I agree with this. Honestly, I'd like it to be also a doctest and if we enable `--doctest-modules` in `pytest` it should not raise any exceptions, which means it needs a more complete example here.